src/components/admin/adminGoogleSheetsPage.js

```diff
@@ -0,0 +1,131 @@
+function renderAdminGoogleSheetsPage({ manifest, adminSettings }) {
+    const existingGoogleSheetsName = adminSettings?.userSettings?.googleSheetsName?.value ?? "";
+    const existingGoogleSheetsUrl = adminSettings?.userSettings?.googleSheetsUrl?.value ?? "";
+    const isManaged = adminSettings?.userSettings?.googleSheetsName?.customizable === false;
+    
+    const page = {
+        id: 'adminGoogleSheetsPage',
+        title: "Admin Google Sheets Config",
+        type: 'page',
+        schema: {
+            type: 'object',
+            properties: {
+                warning: {
+                    type: "string",
+                    description: "Admin configuration will override all user settings. Only one sheet can be used at the same time."
+                }
+            }
+        },
+        uiSchema: {
+            warning: {
+                "ui:field": "admonition",
+                "ui:severity": "warning",  // "warning", "info", "error", "success"
+            }
+        },
+        formData: {
+
+        }
+    }
+    
+    if (existingGoogleSheetsName) {
+        page.schema.properties.adminSheetInfoButton = {
```

Comment:
unify the logic, do something like:

page.schema.properties = {...page.schema.properties, {
    adminSheetInfoButton:{},
    forceGoogleSheets:{},
}}